### PR TITLE
fix(colorswatch): fixes clicking on color did not copy to clipboard

### PR DESCRIPTION
**Pull request by David-Cohen35**

### CODE REVIEW

File: src/util/storybook-docs/ColorSwatch.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ export interface CopyButtonProps {
 
 const CopyButton = ({ value, copyToClipboard = noop }: CopyButtonProps): JSX.Element => (
   <button
-    onClick={() => copyToClipboard((value = ''))}
+    onClick={() => copyToClipboard(value ? value : '')}
```
Comment (reywright) [inline src/util/storybook-docs/ColorSwatch.tsx:26]: Any ternary that goes

`x ? x : y` should be able to be simplified to `x || y`

**Review by reywright — CHANGES_REQUESTED** [review]

File: src/util/storybook-docs/ColorSwatch.tsx (hunk 1 above)
Comment (David-Cohen35) [reply src/util/storybook-docs/ColorSwatch.tsx]: yeah that is beyond better - pushed the change

**Review by reywright — APPROVED** [review]

File: src/util/storybook-docs/ColorSwatch.tsx (hunk 1 above)
Comment (reywright) [reply src/util/storybook-docs/ColorSwatch.tsx]: Just go into depth, the ternary is saying 

if true ? use this value : else use this value

which is the same thing the or clause does:

use this value || use another value

In general ternaries are better when the expression is not used as one of the values. For instance:

```const width = isNavOpened ? "300px" : "100px"```

Comment (David-Cohen35) [reply src/util/storybook-docs/ColorSwatch.tsx]: ya totally makes sense. This is a lot cleaner and easier to read

**Pull request merged**